src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px $hug $hug;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+
+  transition: all 0.5s ease-in-out;
+
+  &:hover {
+    transform: scale(1.2) ;
+  }
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-primary;
+
+    &:hover {
+      color: $text-color-hover;
+    }
+  }
+
+  &__code {
+    color: $text-color-collateral;
+    margin-top: 4px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: $hug;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-tag {
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color-collateral;
+  }
+
+  &__value-tag {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: block;
+    padding: 11px 70px;
+    margin-top: $hug;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $background-color;
+    background-color: $accent-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+  }
```

Comment:
rewrite it according to declaration order https://9elements.com/css-rule-order/

Reply:
Wrote code according to [CSS order](https://codeguide.academy/html-css.html#grouping), which looks more efficient.